src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointPair.java

```diff
@@ -0,0 +1,476 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import com.google.common.annotations.VisibleForTesting;
+import htsjdk.samtools.util.SequenceUtil;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import scala.Tuple2;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Objects;
+
+import static org.broadinstitute.hellbender.tools.spark.sv.BreakpointPair.EvidenceFlankSide.*;
+
+/**
+ * This class represents inferred a pair of breakpoints of a simple SV event
+ * (a simple rearrangement between two genomic locations)
+ * that is suggested by the input {@link ChimericAlignment},
+ * and complications in pinning down the breakpoints's location to exact base pair resolution.
+ */
+@DefaultSerializer(BreakpointPair.Serializer.class)
+class BreakpointPair {
```

Comment:
Maybe we are defining breakpoints differently. I was referring to breakpoints as two genomic locations that are rearranged together in the sample that is different from the reference's arrangement, e.g. for simple inversion the left and right end of the inverted sequence. Hence for simple variants that we are currently dealing with, it always comes in pairs. For insertions, the `first` and `second` of the pair point to the same location.